lib/network/modules/components/nodes/shapes/Image.js

```diff
@@ -58,8 +58,15 @@ class Image extends CircleImageBase {
     ctx.save();
     this.switchImages(selected);
     this.resize();
-    this.left = x - this.width / 2;
-    this.top = y - this.height / 2;
+
+    if(this.options.shapeProperties.useImageCoords){
+      this.left = x;
+      this.top = y;
+    }
+    else{
+      this.left = x - this.width / 2;
+      this.top = y - this.height / 2;
+    }
```

Comment:
Could you format this better, please? Ideally like [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACdqCWAzAFDAC0wGcA6CABxk2jOIIEMK4AFAJ0rjerjIFdicAJIBbBgHM4AYQgQ2AE2IBKYAB0oGDIRKkANnGwxUAXlQAPANzrN6bWRiUTqAJ5WNGAL7WMcXYLXumnZ6BkamZqgAtKjBAO6Y8oSoAPSoAExuNjFE9o6mzlHZOgRwmOIERqkZ3ugeIAA0IJTUtMigDGwcsSwdCMTIIAy6sQzO-Y0ARmwMYADWcDAAyhQzmFDiyDBsfHCNFSK6AOpE8MQrYHCLfZjUAG43zgNgxOMga4Lc7BJiyNhDgo0AFbEMwAIWmcwWiwYIjgABk1nBfv9diBgWZFmtxPoAIp8CDwZF+VErNgfAYTBgTXwNEAUNhrGCHBKEZAADgADI16RBBIdphQBvTeFxbkjGgBHfHwdiUfooBjESJQOBweRq2lsOBSzBar7iH5IP7ExqCESYTbbVHELG46VIo0oxowKnMxIEZBpZ3TTC6LEyESGkC8ACstIEcAAKlT5caASBbjshFB1bBFmAGVQAIIpxYwZz6ImCDweIA).